science/cdo/Portfile

```diff
@@ -35,15 +35,15 @@ depends_lib                 port:netcdf \
                             port:hdf5 \
                             port:curl \
                             port:udunits2 \
-                            port:proj \
+                            port:proj7 \
                             port:fftw-3
```

Comment:
Since you are changing the dependencies for `cdo`, the port will require a rebuild, which normally gets done with the `cdo`'s version changes.

Since we are changing the dependencies while the version stays the same, you will need to increment the revision.

Please edit this commit and add:
```
revision     1
```